src/action/channel.js

```diff
@@ -166,6 +166,32 @@ class ChannelAction {
     }
   }
 
+  /**
+   * List the closed channels by calling the respective grpc api and updating
+   * the closed channels array in the global store.
+   * @return {Promise<undefined>}
+   */
+  async getClosedChannels() {
+    try {
+      const { channels: closedChannels } = await this._grpc.sendCommand(
+        'closedChannels'
+      );
+      this._store.closedChannels = closedChannels.map(channel => ({
+        remotePubkey: channel.remote_pubkey,
+        capacity: parseSat(channel.capacity),
+        channelPoint: channel.channel_point,
+        fundingTxId: this._parseChannelPoint(channel.channel_point)
+          .funding_txid_str,
+        localBalance: parseSat(channel.settled_balance),
+        remoteBalance:
+          parseSat(channel.capacity) - parseSat(channel.settled_balance),
+        status: 'closed',
+      }));
```

Comment:
Don't we want to display the `closingTxid` here as well?

Reply:
True, I changed it so the `details` page shows the closing tx ID for (pending) closed channels.